app/build.gradle

```diff
@@ -24,6 +24,9 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    packagingOptions {
```

Comment:
Any reason why this added?

Reply:
Beacuse there was a build error : `More than one file was found with OS independent path 'META-INF/proguard/androidx-annotations.pro'` . Reference - https://stackoverflow.com/a/52523819